core/modules/file/file.module

```diff
@@ -167,11 +170,13 @@ function file_permission() {
       'title' => t('Manage or replace any file'),
       'description' => t('Allow managing of files from the files overview page.'),
       'restrict access' => TRUE,
+      'warning' => t('Allows people to replace a previously-uploaded file with a new one.'),
     ),
     'delete files' => array(
       'title' => t('Delete any file'),
       'description' => t('Allow deleting files directly from the manage files overview page.'),
       'restrict access' => TRUE,
+      'warning' => t('Allows for potential data loss.'),
```

Comment:
Wording sounds odd to me (although English is not my main language). Wouldn't something like this be better?:

```suggestion
      'warning' => t('Allow actions that may cause accidental data loss.'),
```

Reply:
Updated with my own different wording.